quickstarts/java/gcp-pubsub/config.yml

```diff
@@ -0,0 +1,34 @@
+name: gcp-pubsub
+title: Google Cloud Pub/Sub Java Agent Extension
+description: |
+  # What is Pub/Sub?
+
+  Messaging and ingestion for event-driven systems and streaming analytics. 
```

Comment:
I'd suggest some markdown bullets here to break up this text.

Reply:
Fixed